include/alpaka/mem/view/Accessor.hpp

```diff
@@ -131,28 +131,28 @@ namespace alpaka::experimental
     //! Constrains an existing accessor to the specified access modes.
     // constraining accessor to the same access mode again just passes through
     template<typename TNewAccessMode, typename TMemoryHandle, typename TElem, typename TBufferIdx, std::size_t TDim>
-    ALPAKA_FN_HOST_ACC auto accessWith(const Accessor<TMemoryHandle, TElem, TBufferIdx, TDim, TNewAccessMode>& acc)
+    ALPAKA_FN_HOST auto accessWith(const Accessor<TMemoryHandle, TElem, TBufferIdx, TDim, TNewAccessMode>& acc)
     {
         return acc;
     }
```

Comment:
This is a breaking change, because we could constrain an accessor in-kernel before. That is `accessWith` is allowed to be called inside a kernel.

Reply:
I can revert it that's no problem as long as alpak is not throwing the warning.

Reply:
Mhh, `accessWith` is the reason for the warning. :-( It is not easy possible to use `ALPAKA_FN_HOST_ACC`

Reply:
You can go ahead with your changes. I am hoping to kill those accessors at some point with `mdspan` and pointer support in the SYCL backend.

Reply:
No problem postponing these changes a week more and discussing these.
I saw this warning and thought I need to fix it now or I will forget it until I will see the warning in a half year again.